tests/init.rs

```diff
@@ -0,0 +1,22 @@
+#![feature(plugin)]
+#![cfg_attr(test, plugin(stainless))]
+
+
+extern crate taxonomy_mock as taxonomy;
+extern crate openzwave_adapter;
+
+#[cfg(test)]
+describe! library_integration_test {
+    before_each {
+        use openzwave_adapter::OpenzwaveAdapter;
+        use taxonomy::adapter::AdapterManagerHandleImpl;
+
+        use std::sync::Arc;
+    }
+
+    it "should init" {
```

Comment:
It's the same exact test [as here](https://github.com/fxbox/openzwave-adapter/pull/1/files#diff-b4aea3e418ccdb71239b96952d9cddb6R427), but it enabled to [highlight this issue](https://github.com/fxbox/openzwave-adapter/pull/1/files#r58723246).